deploy/ray-operator/api/v1alpha1/raycluster_types.go

```diff
@@ -86,21 +86,30 @@ type Extension struct {
 	// Pod docker image
 	Image string `json:"image,omitempty"`
 
+	//create operation should have no idList (generate by controller)
+	IdList []string `json:"idList"`
+
```

Comment:
Bump the CRD version if we upgrade the struct

Reply:
This field added as optional for users, do we also update the version?

Reply:
I also agree this is necessary